src/main/resources/ca/corefacility/bioinformatics/irida/database/changesets/22.03/add-sampleName-index-to-sample.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+	<changeSet id="add-sampleName-index-to-sample" author="eric">
+		<createIndex tableName="sample" indexName="IDX_SAMPLE_SAMPLENAME">
```

Comment:
So it looks like this breaks some stuff. If we add this changeset and try to manually load in the required-data.sql file it crashes with a weird and totally unrelated error

`bash: required-data.sql: line 2: syntax error near unexpected token `('
bash: required-data.sql: line 2: `INSERT INTO user (`createdDate`, `modifiedDate`, `email`, `firstName`, `lastName`, `locale`, `password`, `phoneNumber`, `username`, `enabled`, `system_role`, `credentialsNonExpired`) VALUES (now(), now() , 'franklin.bristow@phac-aspc.gc.ca', 'Franklin', 'Bristow', 'en', '$2a$10$yvzFLxWA9m2wNQmHpJtWT.MRZv8qV8Mo3EMB6HTkDnUbi9aBrbWWW', '7029', 'fbristow', 1, 'ROLE_USER', 1);'`

However removing the createIndex section and replacing it with this fixes it

`<sql>
CREATE UNIQUE INDEX IDX_SAMPLE_SAMPLE_NAME ON  sample(sampleName);
</sql>`

I had come across a similar issue a while back when adding in the email pipeline result stuff and ended up fixing it in the same manner.

Reply:
Thanks for taking a look and getting that fixed up